src/preparation.jl

```diff
@@ -84,8 +91,9 @@ function RT.get_chunks(chunker::DocParserChunker, url::AbstractString;
     ## roll up chunks by SEP splitter, then remove it later
     for (doc, src) in zip(docs_, sources_)
         ## roll up chunks by SEP splitter, then remove it later
-        doc_chunks = PT.recursive_splitter(doc, [SEP, separators...]; max_length) .|>
+        doc_chunks = PT.recursive_splitter(doc, [SEP, separators...]; max_length=max_chunk_size) .|>
                      x -> replace(x, SEP => " ") .|> strip |> x -> filter(!isempty, x)
+        chunk_lengths = length.(doc_chunks)
```

Comment:
this is not used?

Reply:
Had put it in for testing. Forgot to remove